autogen/agentchat/contrib/vectordb/mongodb.py

```diff
@@ -276,15 +284,15 @@ def insert_docs(
         For large numbers of Documents, insertion is performed in batches.
 
         Args:
-            docs: List[Document] | A list of documents. Each document is a TypedDict `Document`.
+            docs: List[MongoDocument] | A list of documents. Each document is a TypedDict `MongoDocument`, which may contain ID.
```

Comment:
Would emphasizing "which may not contain an ID" be better?

Reply:
Either, or. My comment was to read as "unlikely to contain", but that is biased towards my current project.

Thoughts on: "which may contain an ID. Documents without ID's will have them generated."